src/Schema/Tables/Medications/Medications.sql

```diff
@@ -0,0 +1,20 @@
+﻿CREATE TABLE [dbo].[Medications]
+(
+    [MedicationId] BIGINT NOT NULL IDENTITY(1, 1),
+    [Name]         NVARCHAR(100) NOT NULL,
+    [LatinName]    NVARCHAR(100) NOT NULL,
+    [Type]         NVARCHAR(50) NOT NULL,
+    [Dose]         SMALLINT NOT NULL,
+    [Direction]    NVARCHAR(50) NOT NULL,
+    [Instruction]  NVARCHAR(max) NOT NULL,
+    [IsDeleted]    BIT NOT NULL DEFAULT 0,
+
+    CONSTRAINT PK_Medications
```

Comment:
We must stick to one style. We write all constraints in one line

Reply:
Ok, agree